net/speedtestcpp/Makefile

```diff
@@ -0,0 +1,66 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=speedtestcpp
+PKG_VERSION:=1.20.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/oskarirauta/speedtestcpp/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=3ab42f95eae91614e1e8455bda1d420a60818fbbd9e975244aaaf9caedf96854
+
+PKG_MAINTAINER:=Oskari Rauta <oskari.rauta@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libspeedtestcpp
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=library for ookla's speedtest
+  DEPENDS:=+libcurl +libstdcpp
+  URL:=https://github.com/oskarirauta/speedtestcpp
+endef
+
+define Package/libspeedtestcpp/description
+  shared library that provides support for ookla's speedtest
+endef
+
+define Package/speedtestcpp
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=SpeedTest++
+  DEPENDS:=+libspeedtestcpp +libstdcpp
+  URL:=https://github.com/oskarirauta/speedtestcpp
+  CONFLICTS:=speedtestpp
```

Comment:
As it was pointed by @1715173329, your forked repository has a few features than original repository. There should be ``PROVIDES``, not ``CONFLICTS``.

Reply:
Or should there be no PROVIDES and CONFLICTS both at all?